src/custom/state/price/updater.ts

```diff
@@ -117,7 +117,7 @@ export default function FeesUpdater(): null {
   // useful to force debounce value to refresh
   const forceUpdateRef = independentField
 
-  const setLoadingQuote = useSetLoadingQuote()
+  const { setNewQuoteLoad, setRefreshQuoteLoad } = useQuoteDispatchers()
```

Comment:
uberniting: but is it correct to say `setNewQuoteLoad` instrad of `setNewQuoteLoading`?

Reply:
done in: https://github.com/gnosis/cowswap/pull/738/commits/838f85789f14d0c9e0bbf4a92285b55a3ba80c5e